app/views/layouts/_sidebar_nav_link.html.erb

```diff
@@ -1,12 +1,12 @@
 <% if !link.display_only_when_signed_in || (link.display_only_when_signed_in && user_signed_in?) %>
   <li>
-    <a href="<%= link.url %>" class="sidebar-navigation-link crayons-link crayons-link--block">
-      <span class="crayons-icon crayons-icon--default">
+    <a href="<%= link.url %>" class="sidebar-navigation-link c-link c-link--block c-link--icon-left">
+      <span class="c-link__icon">
         <%= link.icon.html_safe %>
       </span>
       <%= t("views.main.nav_name.#{link.name}", default: link.name) %>
       <% if link.url.include?("readinglist") %>
-        <span id="reading-list-count" class="crayons-indicator"></span>
+        <span id="reading-list-count" class="crayons-indicator crayons-indicator--accent ml-2 self-center"></span>
```

Comment:
Just double checking this change is intentional, as we've changed the background of this count from gray to the brand color

Before
![reading list with count on a grey background](https://user-images.githubusercontent.com/20773163/146336068-bb39d58e-005a-4329-9f2a-02d731c4fdec.png)

After
![reading list with count on a purple background](https://user-images.githubusercontent.com/20773163/146336082-8257886e-715b-493c-8112-738891c68546.png)

Reply:
So I did intentionally make that change because grey indicator just looked off on hover. But since you're bringing this up I think changing it to accent color might be too jarring. I'm now thinking about replacing it with just classic `(4)`  (literally number in brackets) - so it's not as jarring as the accent one but looks better on hover than default indicator. Thoughts?

Reply:
when it's in the branded it colour it looks more like a notification to me... which doesn't seem right for this situation. I think just popping it in brackets would be good yeah

Reply:
Turns out this thing has been broken on desktop for quite some time already :D So I'll just keep the grey indicator as it was (since that hover style isn't really that jarring on mobile) and will try to address fixing it on desktop in a separate PR some day :)